datacommons/examples/populations_pandas.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""How to use get_populations and get_observations with a Pandas DataFrame.
+
+"""
+
+import datacommons as dc
+import pandas as pd
+
+pd.options.display.max_rows = 10
+pd.options.display.max_colwidth = 30
+
+def print_header(label):
```

Comment:
Seems like we have print_header in every example py file. Could it be extracted out to a util lib?

Reply:
Yeah this sounds good.